libfreerdp/core/input.c

```diff
@@ -206,11 +228,13 @@ void input_send_fastpath_keyboard_event(rdpInput* input, UINT16 flags, UINT16 co
 	eventFlags |= (flags & KBD_FLAGS_RELEASE) ? FASTPATH_INPUT_KBDFLAGS_RELEASE : 0;
 	eventFlags |= (flags & KBD_FLAGS_EXTENDED) ? FASTPATH_INPUT_KBDFLAGS_EXTENDED : 0;
 	s = fastpath_input_pdu_init(rdp->fastpath, eventFlags, FASTPATH_INPUT_EVENT_SCANCODE);
+	if (!s)
```

Comment:
fastpath_input_pdu_init_header returns a stream with 256 byte or FALSE. We only write 2 bytes plus headers here.